Tasks/Maven/analysisResult.ts

```diff
@@ -0,0 +1,8 @@
+// Data class for return from code analysis tools.
+export class AnalysisResult {
+    toolName: string;
+    filesWithViolations: number;
+    totalViolations: number;
+    xmlFilePath: string;
+    htmlFilePath: string;
+}
```

Comment:
An data class, used only once in this PR but expected to be reused for future work on FindBugs and Checkstyle.